conf/templates/mxv/conf-notes.txt

```diff
@@ -0,0 +1,5 @@
+Host Mobility targets are:
+    1. console-hostmobility-image *console only*
+    2. mobility-image-mxv-rig-stripped *MXV production-rig image*
```

Comment:
Why do you call it stripped?

Reply:
OK, but then I suggest using the suffix `console` instead which is more in line with existing naming of images.